docs/_includes/toggle-button.html

```diff
@@ -0,0 +1,254 @@
+<div id="toggle-button">
+    {% include section-header.html name="toggle-button" version=page.versions.toggle-button %}
+
+    <p>The Toggle Button is a button that can be toggled/pressed on or off. It is an atomic piece of the Toggle Button Group. Titles are required and other elements, such as subtitles and images can be included as well.</p>
```

Comment:
For clarity, the instructions around titles can probably be moved to its own line, and then we should call out the fact the these buttons should not contain any structural elements like headings or lists.